vip-real-time-collaboration.php

```diff
@@ -51,11 +52,23 @@
 		return;
 	}
 
+	// Initialize the core classes needed to run the barebones plugin, even if RTC is disabled.
```

Comment:
Perhaps a bit clearer?
```suggestion
	// Initialize the core classes needed manage the Gutenberg experiment.
```